lightly/utils/bounding_box.py

```diff
@@ -31,17 +31,26 @@ class BoundingBox:
         >>> # (x0, y0, x1, y1) = (10, 20, 30, 40)
         >>> W, H = 100, 100 # get image shape
         >>> bbox = BoundingBox(10 / W, 20 / H, 30 / W, 40 / H)
-
     """
 
     def __init__(
         self, x0: float, y0: float, x1: float, y1: float, clip_values: bool = True
     ):
+        """Initializes a BoundingBox object.
+
+        Args:
+            x0: 
+                x0 coordinate.
```

Comment:
I would make it clear that these are relative coordinates and not e.g. numbers of pixels. Also applies to the 3 other arguments.

```suggestion
                x0 coordinate relative to image width.
```